[2024-07-20T21:14:14.239+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-20T21:14:14.264+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: init_ods_layer.get_source_params manual__2024-07-20T21:14:11.059282+00:00 [queued]>
[2024-07-20T21:14:14.272+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: init_ods_layer.get_source_params manual__2024-07-20T21:14:11.059282+00:00 [queued]>
[2024-07-20T21:14:14.273+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-20T21:14:14.282+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_source_params> on 2024-07-20 21:14:11.059282+00:00
[2024-07-20T21:14:14.292+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2935) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-20T21:14:14.293+0000] {standard_task_runner.py:63} INFO - Started process 2937 to run task
[2024-07-20T21:14:14.294+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'init_ods_layer', 'get_source_params', 'manual__2024-07-20T21:14:11.059282+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/init_ods.py', '--cfg-path', '/tmp/tmpe2imhd3i']
[2024-07-20T21:14:14.308+0000] {standard_task_runner.py:91} INFO - Job 242: Subtask get_source_params
[2024-07-20T21:14:14.463+0000] {task_command.py:426} INFO - Running <TaskInstance: init_ods_layer.get_source_params manual__2024-07-20T21:14:11.059282+00:00 [running]> on host 89202c950a25
[2024-07-20T21:14:14.755+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='init_ods_layer' AIRFLOW_CTX_TASK_ID='get_source_params' AIRFLOW_CTX_EXECUTION_DATE='2024-07-20T21:14:11.059282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-20T21:14:11.059282+00:00'
[2024-07-20T21:14:14.766+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-20T21:14:14.785+0000] {base.py:84} INFO - Using connection ID 'source_db' for task execution.
[2024-07-20T21:14:14.914+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-20T21:14:14.915+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-20T21:14:14.922+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=init_ods_layer, task_id=get_source_params, run_id=manual__2024-07-20T21:14:11.059282+00:00, execution_date=20240720T211411, start_date=20240720T211414, end_date=20240720T211414
[2024-07-20T21:14:14.944+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-20T21:14:14.976+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-20T21:14:14.978+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
